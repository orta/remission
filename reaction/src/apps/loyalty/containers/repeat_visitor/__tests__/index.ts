import * as ReactTestUtils from "react-dom/test-utils"
import * as renderer from "react-test-renderer"

import RepeatVisitor from "../index"

describe("RepeatVisitor", () => {
  it("renders the snapshot", () => {
    const repeatVisitor = renderer.create(RepeatVisitor({}))
    expect(repeatVisitor).toMatchSnapshot()
  })

  it("renders the correct content", () => {
    const repeatVisitor: any = ReactTestUtils.renderIntoDocument(RepeatVisitor({}))
    const divTags = ReactTestUtils.scryRenderedDOMComponentsWithTag(repeatVisitor, "div")
    const titleTag = divTags.find(tag => tag.textContent === "Your purchases are being reviewed")
    expect(titleTag).toBeTruthy()
  })
})
