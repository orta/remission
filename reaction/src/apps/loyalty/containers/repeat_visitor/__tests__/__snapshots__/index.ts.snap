// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepeatVisitor renders the snapshot 1`] = `
<div
  className="file__Container-s8sp4hp-0 gLSdEp"
>
  <div
    className="file__Nav-s1w500ep-0 krvdox"
  >
    <a
      className="file__NavIcon-s1w500ep-1 eNnazU"
      href={undefined}
    >
      <div
        className="file-s1ekx40u-0 krLqUx"
        style={undefined}
      >
        î˜ˆ
      </div>
    </a>
    <a
      className="file__StyledLink-mgasw5-0 dIJcvg"
      href="/"
    >
      Back To Artsy
    </a>
  </div>
  <div
    className="file__StyledTitle-s3pz4jh-0 izQygJ"
  >
    Your purchases are being reviewed
  </div>
  <div
    className="file__Section-s8sp4hp-1 ehXvzZ"
  >
    <p
      className="file__Text-s4rtu63-0 eEhqjQ"
    >
      We will be in touch once we confirm your
      <br />
      purchases with the galleries.
    </p>
  </div>
  <div
    className="file__ButtonWrapper-s8sp4hp-2 lpPHz"
  >
    <a
      className="file__InvertedButton-s2qkmvo-0 MNMDa file__StyledButton-muis26-0 exlWFO"
      href="/"
    >
      <span>
        Back to Artsy
      </span>
    </a>
  </div>
</div>
`;
